cpp/test/Ice/background/AllTests.cpp

```diff
@@ -206,7 +124,7 @@ allTests(TestHelper* helper)
         configuration->buffered(true);
         backgroundController->buffered(true);
         background->opAsync();
-        background->ice_getCachedConnection()->close(Ice::ConnectionClose::Forcefully);
+        background->ice_getConnection()->close(Ice::ConnectionClose::Forcefully);
```

Comment:
On Windows, I got a few failures here. The call to `ice_getCachedConnection` was returning `nullptr`, which seems reasonable if the connection was not created by the time `opAsync` return.

Reply:
Sounds like something is going on, perhaps with the `RequestHandlerCache` change? Because the Ice connection is already established, the `opAsync` call should in theory always find the connection in the `OutgoingConnectionFactory` and set it right away...

Reply:
Maybe related to the removal of the other section locator/router, which create the connection?